server/api.js

```diff
@@ -190,20 +190,25 @@ router.post("/ablitiy", authorization, async (req, res) => {
   }
 });
 //<------Delete end point from learning objective------>
-router.delete("/learningobjective/:id", (req, res) => {
+router.delete("/learningobjectives/:id", (req, res) => {
```

Comment:
Don't forget to check that the request is authenticated, AND that the request is coming from a mentor, not a student!